behave/features/constants.py

```diff
@@ -1,5 +1,6 @@
 import os
 
+MAX_TRIES = 2
```

Comment:
@jogilder do we still need this constant?

Reply:
Nope, not any more, I will remove it.